oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/components/Error.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Message, Icon } from "semantic-ui-react";
+import PropTypes from "prop-types";
+
+export const ErrorComponent = ({ error }) => {
```

Comment:
I think this should also be resolved after API is resolved, because there is already an Error component in oarepo ui (in Search folder), but I am not sure yet in which shape the error would come.